tests/integration/test_screenshots_in_docs.py

```diff
@@ -0,0 +1,31 @@
+import os
+import re
+from pathlib import Path
+
+from django.conf import settings
+
+
+SCREENSHOT_RE = re.compile(r"\b[\w-]+.png\b")
```

Comment:
nit - I know this is unlikely to happen, but this regex wouldn't match files like `file.thing.png` or `file(1).png`.

Reply:
Hmm yeah - I was checking for an initial `/` and thought `\b` might be better, but it wouldn't get dotted filenames, and I hadn't thought about brackets in filenames. Have updated the regex now and added some tests for it.